app/DoctrineMigrations/Version20250114155816.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250114155816 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE task RENAME COLUMN co2_emissions TO emitted_co2');
+        $this->addSql('ALTER TABLE task RENAME COLUMN distance_from_previous TO traveled_distance_meter');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+
```

Comment:
Not super critical here, but it's good to keep `down` migration if we need to revert

Reply:
added